packages/databyss-ui/Editor/EditorLine.js

```diff
@@ -37,7 +37,13 @@ const EditorLine = ({ children, onStateChange, value }) => {
   }
 
   // should only have 1 child (e.g. draft/ContentEditable or slate/ContentEditable)
+
+  const _css = children.props.css ? [...children.props.css] : null
+
+  // TODO: CSS GETS OVERRIDDEN HERE
   return React.cloneElement(React.Children.only(children), {
+    ...children.props,
+    _css,
```

Comment:
I encountered this problem with the `css` prop in another context, so it's not isolated to this component tree. Let's use this standard for props:
`overrideCss`: css style object that replaces the `css` prop on the child
`appendCss`: css style object or array of objects that get appended to the css or css array on the child

so an implementation on the child would look like:
```jsx
const _css = { position: 'absolute' }
const Modal = ({
  children,
  overrideCss,
  concatCss,
  ...others
}) => (
  <StyledReactModal
    css={overrideCss || [_css].concat(concatCss)}
    {...others}
  >
    {children}
  </StyledReactModal>
)
```